jupyterlab_bigquery/jupyterlab_bigquery/pagedAPI_handler/pagedQuery_handler.py

```diff
@@ -27,7 +29,8 @@ def __init__(self, application, request, **kwargs):
     self.pool = Pool(NUM_THREADS)
 
     if PagedQueryHandler.client is None:
-      PagedQueryHandler.client = bigquery.Client()
+      PagedQueryHandler.client = bigquery.Client(client_info=ClientInfo(
+          user_agent='jupyterlab_gcpextension/{}'.format(VERSION)))
```

Comment:
Also take away the `{}'.format(VERSION)`. Just use `bigquery.Client(client_info=ClientInfo(user_agent='jupyterlab_gcpextension')`